src/main/java/module/transfer/XMLParser.java

```diff
@@ -44,14 +44,15 @@ public static List<PlayerTransfer> getAllPlayerTransfers(int playerId) {
 
     	final String xml = MyConnector.instance().getTransfersForPlayer(playerId);
         final List<PlayerTransfer> transferList = new Vector<>();
+        if (xml.isEmpty()) return transferList;
 
         final Document doc = XMLManager.parseString(xml);
+        if (doc == null) return transferList;
```

Comment:
I suggest to always use curly braces and never omit them.
I think you get better readability and it is less error prone.

Reply:
Changed